README.rst

```diff
@@ -29,13 +29,206 @@
+service* to every interested library or organisation. But the software can be
+deployed by other institutions, in different contexts as well, if they can
+afford the required effort on configuration or development. It could be run for
+a standalone library in an instance with only one organisation with a single
+library, but RERO ILS is not designed for that use case. It’s powerful, thus
+complex.
+
+What does RERO ILS?
+---------------------
+
+RERO ILS is still under heavy development, but it already provides a *public
+interface* for visitors and patrons. This interface mainly includes searching
+features within the catalog, along with a set of other patron-oriented
+functionalities. It has multiple views, the *union catalog* view, with all the
+documents of all the organisations, and a view for each organisation with all
+the documents belonging to one of its libraries. Patrons are able to place
+requests on desired items, check their profile, see the list of borrowed items
+and so on.
+
+Librarians use a *professional interface*, in which they perform their daily
+tasks:
+
+-  Administering organisations, libraries, circulation policies.
+-  Managing users: patrons, but also librarians and system librarians.
+-  Performing circulation tasks, such as checkins, checkouts, renewals,
+   managing requests.
+-  Managing patron fees (overdue, subscriptions).
+-  Searching for documents, describing documents, linking authorities to
+   documents, importing metadata in MARC format, adding items to
+   documents.
+-  Setting serials patterns, receiving issues.
+-  Managing acquisition budgets, vendor information, orders and invoices.
+
+The first major release of RERO ILS (``v1.0.0``) is planned for the end of 2020,
+and to be live in production in the first half of 2021, with the minimal
+features needed to run real world networks. Then, the development of RERO ILS
+will go on to reach full ILS functions, but it will never ends, as long as a
+software can always be improved and adapted to the need of its users.
+
+How to install RERO ILS?
+------------------------
+
+The installation process is described in a `specific file`_.
+
+For a development environment you can check another
+`documentation`_.
+
+Check the `ecosystem`_ section.
+
+How to contribute to RERO ILS?
+------------------------------
+
+You can test the latest release on
+`ils.test.rero.ch`_.
+
+If you have questions, you can ask the development team on `Gitter`_.
+
+You can also `open an issue`_.
+
+To contribute to the code itself, please check the
+`contributing recommandations`_.
+
+The RERO ILS ecosystem
+----------------------
+
+Three GitHub repositories for RERO ILS
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The `rero-ils GitHub project`_ contains the main projet for RERO ILS, basically
+providing the backend. To work on the frontend of the project, you also need
+`rero-ils-ui`_, which is based on `ng-core`_, MEF, and RERO EBOOKS (see the
+next two section).
```

Comment:
There's something wrong in this sentence, because actually you don't need MEF and EBOOKS to work on the frontend.